Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,49 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+/// <summary>
+/// Controls the movement of a player object. Adds gravity and handles move inputs.
+/// </summary>
+[RequireComponent(typeof(CharacterController))]
+public class PlayerMovement : MonoBehaviour
+{
+    [SerializeField] private float moveSpeed;
```

Comment:
Good approach to public/private variables. Like mentioned in the Code Standards, this keeps variables protected whilst allowing developers to change them outside of the script.